lib/elixir/pages/getting-started/debugging.md

```diff
@@ -174,7 +174,7 @@ We have just scratched the surface of what the Erlang VM has to offer, for examp
 
   * [Microstate accounting](http://www.erlang.org/doc/man/msacc.html) measures how much time the runtime spends in several low-level tasks in a short time interval
 
-  * Mix ships with many tasks under the `profile` namespace, such as `cprof` and `fprof`
+  * Mix ships with many tasks under the `profile` namespace, such as [`cprof`](`Mix.Tasks.Profile.Cprof`) and [`fprof`](`Mix.Tasks.Profile.Fprof`)
```

Comment:
```suggestion
  * Mix ships with many tasks under the `profile` namespace, such as `mix cprof` and `mix fprof`
```

Reply:
Small amendment to fix the links: [2fccb1a](https://github.com/elixir-lang/elixir/pull/13290/commits/2fccb1a08925677723db8b9b3ce1517c14e0041d)